DOCS/man/options.rst

```diff
@@ -3481,7 +3481,7 @@ Window
 Disc Devices
 ------------
 
-``--cdrom-device=<path>``
+``--cdda-device=<path>``
     Specify the CD-ROM device (default: ``/dev/cdrom``).
```

Comment:
not sure what the most accurate terminology is here
```suggestion
    Specify the CD device for CDDA playback (default: ``/dev/cdrom``).
```

Reply:
Sounds good to me.